dragonfly/block/ice.go

```diff
@@ -0,0 +1,44 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item/tool"
+
+// TODO: Slipperiness, melting and SilkTouch
```

Comment:
Melting should be implementable using the current system. From the wiki:
"Ice can be used to create water either by its melting or being broken. If there is another block directly underneath the ice block, the ice reverts to water when broken. Ice also melts into water if the light level immediately next to it on any side is higher than 11, from light sources other than sunlight (and regardless of whether there's a block below)."